frontend/lib/api/election/ElectionStatusProvider.tsx

```diff
@@ -20,7 +22,7 @@ export function ElectionStatusProvider({ children, electionId }: ElectionStatusP
   });
 
   if (error) {
-    throw new Error("Could not fetch election statuses");
+    throw new NotFoundError("Could not fetch election statuses");
```

Comment:
Shall we make this error message in Dutch as well?

Reply:
I was planning to properly translate with the i18n PR - to prevent duplicate work.